app/controllers/idv/cancellations_controller.rb

```diff
@@ -83,11 +82,6 @@ def location_params
       params.permit(:step, :location).to_h.symbolize_keys
     end
 
-    def reset_doc_auth
-      user_session.delete('idv/doc_auth')
-      user_session['idv'] = {}
```

Comment:
We cleared idv_session above in `cancel_session` where this is called, and user_session['idv'] is not a hash (it was in the FSM), and also we use user_session[:idv] everywhere else.